### PR TITLE
Feature/search functionality

### PR DESCRIPTION
**Pull request by ShawnBatson**
## Description ##
This branch deals with the search bar, the search bar functionality, the search results, and the search results rendering page.  Along with the main feature, I Implemented various small design corrections.
## Type ##

- [x] This is a bugfix.
- [x] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [x] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [x] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [x] Have you added two reviewers to this pull request?

### CODE REVIEW

**Comment by ShawnBatson** [comment]
[As a pet owner, I want to be able to search for groomers.](https://trello.com/c/ptp6GLJr/4-as-a-pet-owner-i-want-to-be-able-to-search-for-groomers)

**Comment by ShawnBatson** [comment]
[create a feed of cards for each groomer](https://trello.com/c/0kTcjM8n/20-create-a-feed-of-cards-for-each-groomer)

**Comment by ShawnBatson** [comment]
[create a search bar](https://trello.com/c/5jqoblC1/19-create-a-search-bar)

**Comment by ShawnBatson** [comment]
[add place for user to select that groomer (button?) which will re-direct pet owner to Groomer's profile](https://trello.com/c/JOvgjvAP/22-add-place-for-user-to-select-that-groomer-button-which-will-re-direct-pet-owner-to-groomers-profile)

**Comment by ShawnBatson** [comment]
[search by location - zip code/city, state](https://trello.com/c/x52o2ZWG/21-search-by-location-zip-code-city-state)

File: src/components/pages/SearchResults/SearchResultsPage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (jessicagertig) [inline src/components/pages/SearchResults/SearchResultsPage.js]: For clarity perhaps we should rename this component as SearchResultsCard

**Review by jessicagertig — COMMENTED** [review]
Let's go ahead and delete the RenderSearchBarContainer.js file (this says it's empty).

File: src/components/pages/SearchResults/SearchResultsPage.js (hunk 1 above)
Comment (ShawnBatson) [reply src/components/pages/SearchResults/SearchResultsPage.js]: I would normally call it "card" but the search bar is rendered there, as well, which is static on the page. If you still want me to change it, though, I will.

Comment (ShawnBatson) [reply src/components/pages/SearchResults/SearchResultsPage.js]: There we go, I see what you were talking about. I fixed the nomenclature of the files and file structure.

File: src/components/pages/Search/RenderSearchPage.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const [allGroomers, setAllGroomers] = useState();
+  const [searchValue, setSearchValue] = useState('');
+  const [filteredGroomers, setFilteredGroomers] = useState([]);
+
+  useEffect(() => {
+    Axios.get('http://localhost:8000/groomers')
+      .then(res => {
+        console.log('this is res.data', res.data); // sending back an array of objects
+        setAllGroomers(res.data);
+        // console.log('this is in the useEffect', allGroomers);
```
Comment (jessicagertig) [inline src/components/pages/Search/RenderSearchPage.js]: Let's remove the console logs too before merging.

**Review by jessicagertig — CHANGES_REQUESTED** [review]
Hey overall EXCELLENT job Shawn!  Thanks for taking initiative on styling.  Just one more cleanup, let's remove the console logs (other than the error console log) before we merge to main.

File: src/components/pages/Search/RenderSearchPage.js (hunk 2 above)
Comment (ShawnBatson) [reply src/components/pages/Search/RenderSearchPage.js]: Another good catch.  That one will be hard for me to remember.

Comment (ShawnBatson) [reply src/components/pages/Search/RenderSearchPage.js]: Done

**Review by jessicagertig — DISMISSED** [review]
I see there are several 'unchanged files' with GithHub Actions check annotations.  Since Shawn didn't work on any of those files, let's make a separate branch/pull request to fix all these checks

**Review by jessicagertig — APPROVED** [review]
Thanks for pulling the bug fixes

File: public/index.html [hunk 3]
```diff
@@ -26,7 +26,12 @@
     -->
     <title>Basic SPA</title>
   </head>
-  <body>
+  <body style="  background: linear-gradient(
+    70deg,
+    rgba(255, 255, 255, 1) 0%,
+    rgba(142, 177, 217, 1) 25%,
+    rgba(255, 255, 255, 1) 100%
```
Comment (Broast42) [inline public/index.html:33]: Really like the background colors.

**Review by Broast42 — APPROVED** [review]
Good job keep it up.

**Pull request merged**